releasenotes/notes/enable_target_aware_meas_map-0d8542402a74e9d8.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added an argument ``meas_map`` to :class:`~.Target`.
```

```suggestion
    Added :attr:`~.Target.meas_map` which represents a hardware constraint of qubits
    measured concurrently. This constraint is provided in the nested list form,
    in which each element represents qubit group to be measured together. 
    In an example below,
    
    .. code-block:: python
    
        [[0, 1], [2, 3, 4]]
     
    qubits 0 and 1, and 2, 3 and 4 are measured together on the device.
    This constraint doesn't block measuring an individual qubit,
    however, Qiskit scheduler must consider the alignment of 
    measure operations for those qubits.    
```

You need more description otherwise user cannot understand how to use/provide new constraint.